modules/hardware/nvidia-jetson-orin/jetson-orin.nix

```diff
@@ -14,14 +16,61 @@
       passthrough-patch = ./pci-passthrough-agx-test.patch;
       vfio-pci = "vfio-pci.ids=10ec:c82f";
       deviceTree = "tegra234-p3701-host-passthrough.dtb";
+      buspath = [
+        {
+          bus = "pci";
+          path = "0001:01:00.0";
+        }
+      ];
+      kernelParams = [];
+      wireless = {
+        enable = true;
+        networks."SSID_OF_NETWORK".psk = "WPA_PASSWORD";
```

Comment:
This should not be left enabled (uncommented) as it will try to configure SSID: `SSID_OF_NETWORK` with password: `WPA_PASSWORD`. Please leave commented out or refactor into a WIFI module that can be enabled/disabled.